src/utils/__tests__/getLocationInfoFromSites.test.js

```diff
@@ -164,4 +175,55 @@ describe('getLocationInfoFromSites', () => {
 			getLocationInfoFromSites('approved', nctId, sitesNoUSLocation)
 		).toMatchObject(expectedJSX);
 	});
+
+	test('should return expected string when recruitment status is "Complete"', () => {
```

Comment:
Can you update the test case to reflect recruitment statuses being tested in here.